app/assets/stylesheets/components/_header.scss

```diff
@@ -23,12 +25,16 @@
     @include fluid-size('font-size', 10px, 21px);
 
     &.selected {
-      border-bottom: 5px solid orange;
+      border-bottom: 5px solid darkorange;
```

Comment:
Why are we using orange? Seems like the design file is using the `$c-pink` we setup.